pages/create-release.js

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from 'react'
+import { supabase } from '@/utils/supabase'
+import { useUser } from '@/utils/context/user'
+
+const trueFalse = [
+  { id: 1, text: 'True', value: true },
+  { id: 2, text: 'False', value: false }
+]
+
+const releaseTypes = [
+  { id: 1, text: 'LP' },
+  { id: 2, text: 'EP' },
+  { id: 3, text: 'Single' },
+  { id: 4, text: 'Compilation' },
+  { id: 5, text: 'Soundtrack' },
+  { id: 6, text: 'Choose release type', isDisabled: true }
+]
+
+export default function CreateRelease() {
+  const { activeUser } = useUser()
```

Comment:
I think you should simplify this to `user`

Reply:
I agree, but I ran into an issue and that is why I changed it to activeUser. I can revisit that.